README.md

```diff
@@ -1,24 +1,113 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
 
-Things you may want to cover:
+  <h1>Hello React Back End</h1>
 
-* Ruby version
+</div>
 
-* System dependencies
+# 📗 Table of Contents
 
-* Configuration
+- [📖 About the Project](#about-project)
+  - [🗝️ Key Features](#key-features)
+  - [🛠️ Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Database creation
+# 📖 Hello React Front End <a name="about-project"></a>
 
-* Database initialization
+**Hello React Front End** is the back-end of a project, which is built using both back-end and front-end. The React front-end can be found <a href="https://github.com/AyomipoMajek/hello-rails-frontend" >here</a>.
 
-* How to run the test suite
+## 🗝️ Key Features <a name="key-features"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+- Implements `@reduxjs/toolkit`
+- Implements `react-router-dom`
 
-* Deployment instructions
+## 🛠️ Built With <a name="built-with"></a>
 
-* ...
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://react.dev">React</a></li>
+  </ul>
+</details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Install [Node.js](https://nodejs.org/en)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/AyomipoMajek/hello-rails-backend.git
+  cd hello-react-front-end
+```
+
+### Usage
+
+To run the project, execute the following commands:
+
+```sh
+  npm install
+  npm start
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] [optional] @AyomipoMajek it would be nice if you changed the getting started section from describing how to install and run a frontend app to describe how to run a backend app.